packages/stitch/src/stitchSchemas.ts

```diff
@@ -101,6 +101,7 @@ export function stitchSchemas<TContext extends Record<string, any> = Record<stri
     astNode: schemaDefs.schemaDef,
     extensionASTNodes: schemaDefs.schemaExtensions,
     extensions: null,
+    assumeValid: true,
```

Comment:
Maybe we can let user decide because \`assumeValid\` is probably already available in \`IStitchSchemasOptions\`. So we can use `rest.assumeValid`. What do you think?

Reply:
```typescript
  /**
   * Set to true to assume the SDL is valid.
   *
   * Default: false
   */
  assumeValidSDL?: boolean;
  /**
   * When building a schema from a GraphQL service's introspection result, it
   * might be safe to assume the schema is valid. Set to true to assume the
   * produced schema is valid.
   *
   * Default: false
   */
  assumeValid?: boolean;
   ```
   
   we have even more options to choose from :) ... based on the descriptions, i agree `assumeValid` is more accurate ... also in the rest of the codebase its already provided, like `getStitchingOptionsFromSupergraphSdl`